pulsar-client-cpp/python/src/config.cc

```diff
@@ -110,61 +112,76 @@ class LoggerWrapper: public Logger {
 
    public:
 
-    LoggerWrapper(const std::string &logger, PyObject* pyLogger) {
+    LoggerWrapper(const std::string &filename, PyObject* pyLogger) {
         _pyLogger = pyLogger;
         Py_XINCREF(_pyLogger);
 
+        std::unique_ptr<LoggerFactory> factory(new ConsoleLoggerFactory());
+        fallbackLogger = factory->getLogger(filename);
+
         _updateCurrentPythonLogLevel();
     }
 
-    LoggerWrapper(const LoggerWrapper& other) {
+    LoggerWrapper(LoggerWrapper& other) {
```

Comment:
```suggestion
    LoggerWrapper(const LoggerWrapper& other) {
```

Don't remove the const qualifier. Or do you have any reason to remove it?

Reply:
I removed the whole method to avoid bloating the code unnecessarily and the potential memory leak.